server/src/test/java/io/crate/integrationtests/DiskThresholdDeciderIT.java

```diff
@@ -164,8 +165,8 @@ public void testHighWatermarkNotExceeded() throws Throwable {
             assertThat(getShardRoutings(dataNode0Id, indexName)).isEmpty();
         });
 
-        // increase disk size of node 0 to allow just enough room for one shard, and check that it's rebalanced back
-        fileSystemProvider.getTestFileStore(dataNode0Path).setTotalSpace(minShardSize + WATERMARK_BYTES + 1L);
+        // increase disk size of node 0 to allow enough room for one shard, and check that it's rebalanced back
+        fileSystemProvider.getTestFileStore(dataNode0Path).setTotalSpace(minShardSize + WATERMARK_BYTES + 100L);
```

Comment:
Given that the table stats are also in the data path I had to increase the size here to account for additional space used by the stats.